doc/docs.md

```diff
@@ -167,6 +181,16 @@ fn main() {
 
 Unlike most languages, variable shadowing is not allowed. Declaring a variable with a name that is already used in a parent scope will result in a compilation error.
 
+Variable names in V language should be snake_case style:
```

Comment:
"should be snake_case style" would not be grammatically correct.
"should be written in snake_case" would be better.

Reply:
Fixed along the document